ar_exercise/exercise.rb

```diff
@@ -51,21 +69,51 @@ class Apartment < ActiveRecord::Base
 # Hint, the following methods will help: `new`, `create`, `save`, `uddate`, `destroy`
 
 # Create 3 new apartments, and save them to the DB
+# apt1 = Apartment.new(address: "123 Lollipop Lane", monthly_rent: 3000, sqft: 800, num_beds: 2, num_baths: 2)
+# apt1.save
+# apt2 = Apartment.new(address: "4900 Elm Street", monthly_rent: 2100, sqft: 400, num_beds: 1, num_baths: 1)
+# apt2.save
+# apt3 = Apartment.new(address: "5 Backwoods Place", monthly_rent: 2000, sqft: 5000, num_beds: 22, num_baths: 4)
+# apt3.save
+
 # Create at least 9 new tenants and save them to the DB. (Make sure they belong to an apartment)
+# tenant1 = Tenant.create(name: "Pookins Magoo", age: 45, gender: "Female", apartment_id: 2)
+# tenant2 = Tenant.create(name: "Carrot", age: 2, gender: "Female", apartment_id: 25)
+# tenant3 = Tenant.create(name: "Roderick Lakowski III", age: 67, gender: "Male", apartment_id: 22)
+# tenant4 = Tenant.create(name: "Taco Truck", age: 91, gender: "Female", apartment_id: 21)
+# tenant5 = Tenant.create(name: "Derek Langton", age: 22, gender: "Male", apartment_id: 23)
+# tenant6 = Tenant.create(name: "Falcon Spoonies", age: 58, gender: "Female", apartment_id: 25)
+# tenant7 = Tenant.create(name: "Bart Ulie", age: 12, gender: "Male", apartment_id: 20)
+# tenant8 = Tenant.create(name: "Panko Crumm", age: 27, gender: "Female", apartment_id: 5)
+# tenant9 = Tenant.create(name: "Bleep Bloop", age: 150, gender: "Bleep bloop", apartment_id: 8)
+# tenant10 = Tenant.create(name: "Pinon Errata", age: 74, gender: "Female", apartment_id: 11)
+
 # Note: you'll use this little bit of code as a `seeds.rb` file later on.
 
 # Birthday!
 # It's Kristin Wisoky's birthday. Find her in the DB and change her age to be 1 year older
+# kristin = Tenant.find_by(name: "Kristin Wisoky")
+# puts kristin.name, kristin.age
+# kristin.age = kristin.age + 1
+# puts kristin.name, kristin.age
```

Comment:
Technically this will only update the in-memory copy of the object. We weren't specific, but in this case you'd probably want to call `.save` to save to the DB. Or you can use `.update` or `.increment`.

```
kristin.save